client/my-sites/customer-home/locations/primary/index.jsx

```diff
@@ -7,23 +7,27 @@ import React from 'react';
  * Internal dependencies
  */
 import GoMobile from 'my-sites/customer-home/cards/features/go-mobile';
+import WebinarPromotion from 'my-sites/customer-home/cards/features/webinar-promotion';
 import ChecklistSiteSetup from 'my-sites/customer-home/cards/tasks/checklist-site-setup';
 import MasteringGutenberg from 'my-sites/customer-home/cards/education/mastering-gutenberg';
 import QuickLinks from 'my-sites/customer-home/cards/actions/quick-links';
 import WpForTeamsQuickLinks from 'my-sites/customer-home/cards/actions/wp-for-teams-quick-links';
 import ConnectAccounts from 'my-sites/customer-home/cards/tasks/connect-accounts';
 import FindDomain from 'my-sites/customer-home/cards/tasks/find-domain';
+import Webinar from 'my-sites/customer-home/cards/tasks/webinar';
 import config from 'config';
 
 const cardComponents = {
 	'home-feature-go-mobile-phones': GoMobile,
+	'home-feature-webinar-promotion': WebinarPromotion,
```

Comment:
I think we can drop the wiring here @Aurorum for this PR we should be able to add this in a follow-up once the server side patch lands.

Reply:
Dropped! :)